python/ingestion/cdc_wisqars_utils.py

```diff
@@ -104,17 +104,24 @@ def convert_columns_to_numeric(df: pd.DataFrame, columns_to_convert: List[str]):
         df[column] = pd.to_numeric(df[column], errors='coerce')
 
 
-def generate_cols_map(prefixes, suffix):
+def generate_cols_map(estimated_total_cols: List[str], suffix: str):
     """
-    Generates a mapping of columns with given prefixes to their corresponding columns
-    with specified suffix.
+    Generates a mapping of column names from a list of estimated total column names to a column name
+    with a specified suffix.
```

Comment:
Generates a mapping of column prefixes to new column names, based on incoming list of prefixes and incoming suffix. Note: if any of the incoming prefixes are in the count column format (ending in `_estimated_total`), the resulting dict will use the original count col as the key, but will swap `_estimated_total` for the new suffix in the column name value.